pkg/cmd/extension/http.go

```diff
@@ -13,32 +13,49 @@ import (
 	"github.com/cli/cli/v2/internal/ghrepo"
 )
 
-func hasScript(httpClient *http.Client, repo ghrepo.Interface) (hs bool, err error) {
+func repoExists(httpClient *http.Client, repo ghrepo.Interface) (bool, error) {
```

Comment:
I don't love that this returns a boolean and an error for such a short function. I'm wondering if it should only return an error or nil, but it feels strange to return an error for the lack of existence of a repo 🤔

Reply:
I don't think that it feels odd to return both a boolean and an error. One thing we can do is just return an error and have a nice error message for a 404 response indicating that the repo does not exist. We do something similar in `HasMinimumScopes` in `oauth_scopes.go`.